src/api/api.ts

```diff
@@ -64,7 +82,10 @@ class API {
 
     if (!response.ok) {
       toast.error(data.message || "Wrong server response!");
-      throw new Error(data.message || "Wrong server response!");
+      throw new Error(
+        data.message + `. Status code: ${response.status}` ||
+          "Wrong server response!"
+      );
```

Comment:
Użyłbym tutaj string template'a
```suggestion
      throw new Error(
        `${data.message}. Status code: ${response.status}` ||
          "Wrong server response!"
      );
```